stl/inc/format

```diff
@@ -536,15 +536,20 @@ protected:
     }
 
     _Fmt_codec_base() {
+        thread_local const _Cvtvec _Cvt_cache = [] {

```

Comment:
We can't use `thread_local` in the STL so that we don't use any TLS slots our customers might need.

Reply:
It also interferes with optimizations that Windows wants to perform.